### PR TITLE
Copyedit

### PR DESCRIPTION
**Pull request by ghost**
Make text make sense™

### CODE REVIEW

**Comment by CesiumCs** [comment]
slick

File: rekt.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ func evalRatelimit(author string) bool {
 		rateLimitDuration := RateLimit
 		if author == "162848980647018496" {
 			// berrely is pee pee poo poo
-			rateLimitDuration = 2 * time.Hour
+			rateLimitDuration = 0 * time.Hour
```
Comment (CesiumCs) [inline rekt.go:36]: maybe bump it up in the other direction instead?

**Review by CesiumCs — CHANGES_REQUESTED** [review]

**Comment by DoctorBang** [comment]
Nice. Fully expected this the moment Cesium linked this PR.

File: rekt.go (hunk 1 above)
Comment (ghost) [reply rekt.go:36]: Yeah to -1

Comment (CesiumCs) [reply rekt.go:36]: does bri*ish "maffs" class not teach math

**Pull request closed without merging**